packages/jest-editor-support/src/Runner.js

```diff
@@ -98,6 +106,9 @@ export default class Runner extends EventEmitter {
   }
 
   closeProcess() {
+    if (!this.debugprocess) {
+      return;
+    }
```

Comment:
What is this for?

Reply:
couple of time i got undefined error on this.debugprocess.pid - probably because i called closeProcess() twice ... and forget to revert this code

BTW: i think, that if -> return is better than throw ... specially in this context ...